app/uk/gov/hmrc/taxenrolmentassignmentfrontend/connectors/EACDConnector.scala

```diff
@@ -89,4 +104,27 @@ class EACDConnector @Inject()(httpClient: HttpClient,
       )
   }
 
+  def queryKnownFactsByNinoVerifier(nino: String)(
+    implicit ec: ExecutionContext,
+    hc: HeaderCarrier
+  ): TEAFResult[Option[String]] = EitherT {
+    val url = s"${appConfig.EACD_BASE_URL}/enrolment-store/enrolments"
+    val request = new KnownFactQueryForNINO(nino)
+    httpClient
+      .POST[KnownFactQueryForNINO, HttpResponse](url, request)
+      .map(
+        httpResponse =>
+          httpResponse.status match {
+            case OK =>
+              Right(Some(httpResponse.json.as[KnownFactResponseForNINO].getUTR))
+            case NO_CONTENT => Right(None)
+            case status =>
+              logger.logEvent(
+                logUnexpectedResponseFromEACDQueryKnownFacts(nino, status)
```

Comment:
Can you add the error message we get back into the log? It'll help diagnose why we got the status.

Reply:
done